src/GovUk.Education.ExploreEducationStatistics.Common/Services/Interfaces/IFileTypeService.cs

```diff
@@ -1,3 +1,4 @@
+using System;
```

Comment:
There's some unused imports that could be cleaned up here.

Reply:
Cleaned up thanks